src/store/modules/order/actions.ts

```diff
@@ -103,6 +115,86 @@ const actions: ActionTree<OrderState, RootState> = {
     } catch(error) {
       showToast(translate("Something went wrong"));
     }
+    return resp;
+  },
+
+  async updateAppliedFilters({ state, commit, dispatch }, payload) {
+    if (payload.filterName === 'selectedPoIds') {
+      const poIds = state.query.selectedPoIds;
+      !poIds.includes(payload.value) ? poIds.push(payload.value) : poIds.splice(poIds.indexOf(payload.value), 1)
+      payload.value = poIds
+    }
+    commit(types.ORDER_FILTERS_UPDATED, payload)
+    const resp = await dispatch('updateQuery', { viewSize: process.env.VUE_APP_VIEW_SIZE, viewIndex: 0, queryString: state.query.queryString })
```

Comment:
If queryString is already in state no need to pass it

Reply:
Done Sir